base/scripts/logging.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+echo "*.* @@192.168.66.2:24224" >> /etc/rsyslog.conf
```

Comment:
IIUC you pass all logs via fluent, so you do not need this line

Reply:
Ah, this is early forgotten artefact. Thanks :)